### PR TITLE
Adding tag to kubernetes events for pods, rs, deployments

### PR DESCRIPTION
**Pull request by CharlyF**
### What does this PR do?

A brief description of the change being made with this pull request.

### Motivation

What inspired you to submit this pull request?

### Additional Notes

Anything else we should know when reviewing?

### Describe your test plan

Write there any instructions and details you may have to test your PR.

### CODE REVIEW

**Review by vboulineau — APPROVED** [review]

File: pkg/collector/corechecks/cluster/kubernetes_eventbundle.go [hunk 1]
```diff
@@ -132,6 +132,19 @@ func (b *kubernetesEventBundle) formatEvents(clusterName string, providerIDCache
 	return output, nil
 }
 
+func addKindRelatedTag(kind, name string) string {
```
Comment (ahmed-mez) [inline pkg/collector/corechecks/cluster/kubernetes_eventbundle.go:135]: could we add `Statefulset` ? I'm comparing to what we do in the tagger https://github.com/DataDog/datadog-agent/blob/master/pkg/tagger/collectors/kubelet_extract.go#L112-L122

Comment (vboulineau) [reply pkg/collector/corechecks/cluster/kubernetes_eventbundle.go:135]: Also thinking, what about adding a generic `kube_lower($kind)` tag? Would work quite well (we might still need to add `pod_name` as it's non standard

Comment (ahmed-mez) [reply pkg/collector/corechecks/cluster/kubernetes_eventbundle.go:135]: +1

Comment (CharlyF) [reply pkg/collector/corechecks/cluster/kubernetes_eventbundle.go:135]: ok, the card was just for podname initially, but i will add that

**Review by ahmed-mez — APPROVED** [review]

File: pkg/collector/corechecks/cluster/kubernetes_eventbundle.go (hunk 1 above)
Comment (ahmed-mez) [reply pkg/collector/corechecks/cluster/kubernetes_eventbundle.go:135]: feel free to backlog it since it's out of scope

**Pull request merged**